src/encoded/schemas/individual_zebrafish.json

```diff
@@ -0,0 +1,116 @@
+{
+    "title": "Zebrafish strain",
+    "description": "Schema for submitting a zebrafish cellline/strain.",
+    "id": "/profiles/individual_zebrafish.json",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "required": ["award", "lab"],
+    "identifyingProperties": ["uuid", "accession", "aliases"],
+    "additionalProperties": false,
+    "mixinProperties": [
+        { "$ref": "mixins.json#/schema_version" },
+        { "$ref": "mixins.json#/uuid" },
+        { "$ref": "mixins.json#/status" },
+        { "$ref": "individual.json#/properties" },
+        { "$ref": "mixins.json#/accession" },
+        { "$ref": "mixins.json#/aliases" },
+        { "$ref": "mixins.json#/attribution" },
+        { "$ref": "mixins.json#/submitted" },
+        { "$ref": "mixins.json#/modified" },
+        { "$ref": "mixins.json#/release_dates" },
+        { "$ref": "mixins.json#/documents" },
+        { "$ref": "mixins.json#/tags" },
+        { "$ref": "mixins.json#/notes" },
+        { "$ref": "mixins.json#/static_embeds" }
+    ],
+    "mixinFacets" : [
+        { "$ref": "mixins.json#/facets_common" }
+    ],
+    "dependencies": {
+      "age": [ "age_units"]
+    },
+    "properties": {
+        "schema_version": {
+            "default": "1"
+        },
+        "zebrafish_life_stage": {
```

Comment:
These are appropriate for Drosophila but not so much for Zebrafish.
Top level stages at this link might work better: https://zfin.org/zf_info/zfbook/stages/index.html

Reply:
Agreed, and updated